modules/apps/27-interchain-accounts/types/packet.go

```diff
@@ -63,3 +72,41 @@ func (ct CosmosTx) UnpackInterfaces(unpacker codectypes.AnyUnpacker) error {
 
 	return nil
 }
+
+// GetPacketSender returns the sender address of the interchain accounts packet data.
+// It is obtained from the source port ID by cutting off the ControllerPortPrefix.
+// If the source port ID does not have the ControllerPortPrefix, then an empty string is returned.
+//
+// NOTE:
+//   - The sender address is set by the packet sender and may not have been validated a signature
+//     check if the packet sender isn't the interchain accounts module.
+//   - The sender address must only be used by modules on the sending chain.
+func (InterchainAccountPacketData) GetPacketSender(sourcePortID string) string {
+	found := strings.HasPrefix(sourcePortID, ControllerPortPrefix)
+	if !found {
+		return ""
+	}
+	return strings.TrimPrefix(sourcePortID, ControllerPortPrefix)
```

Comment:
used `HasPrefix` and `TrimPrefix` instead of `CutPrefix` here because `CutPrefix` was added in go1.20